cmd/zc_enumerator.go

```diff
@@ -388,6 +388,8 @@ func initResourceTraverser(resource common.ResourceString, location common.Locat
 			}
 
 			output = newBlobAccountTraverser(resourceURL, *p, *ctx, includeDirectoryStubs, incrementEnumerationCounter)
+		} else if listOfVersionIds != nil {
+			output = newBlobVersionsTraverser(resourceURL, *p, *ctx, recursive, includeDirectoryStubs, incrementEnumerationCounter, listOfVersionIds)
```

Comment:
>output = newBlobVersionsTraverser(resourceURL, *p, *ctx, recursive, includeDirectoryStubs, incrementEnumerationCounter, listOfVersionIds) [](start = 3, length = 137)

Can we enforce that the source is a single blob?

Reply:
and also the destination can only be local or trash

---
In reply to: [465411289](https://github.com/Azure/azure-storage-azcopy/pull/1123#discussion_r465411289) [](ancestors = 465411289)

Reply:
Done.